invenio_rdm_records/records/api.py

```diff
@@ -272,6 +294,24 @@ class RDMDraft(CommonFieldsMixin, Draft):
 RDMFileDraft.record_cls = RDMDraft
 
 
+class RDMDraftMediaFiles(RDMDraft):
+    """RDM Draft media file API."""
+
+    files = FilesField(
```

Comment:
this is the transparency part - service components point to this field, while in fact all the data is stored in the media_files - thanks to the "key" attribute